src/filters/pages.js

```diff
@@ -265,6 +265,20 @@ const pageYears = (collection, events = true) =>
     return page;
   });
 
+/* @docs
+label: eventSort
+category: Sorting
+note: |
+  Sort pages based on either the page date,
+  or the most recently past event date.
+params:
+  collection:
+    type: array
+    note: containing 11ty page objects
+*/
+const eventSort = (collection) =>
+  pageYears(collection).sort((a, b) => a.sort > b.sort);
```

Comment:
It took me an hour to track down this bug... which is that `.sort()` expects a number, not a boolean. 😖 

I find the large number of small filters, many of which call each other, very difficult to follow and debug. I'd love to clean them up with an eye toward clarity of naming, combining where possible, and removing extra ones -- but obviously none of that should hold up merge/launch.

Reply:
I'm very open to that. I'm not a JS expert, and I wrote a lot of these in a hurry. But usually when I split them it's because we ended up needing just one aspect of it somewhere. I'm curious to see how you'd approach it.

Reply:
Yeah, I didn't intend that as "this is bad" as it reads -- the JS is great, it's been built up somewhat rapidly, and could improve with a refactor. 👍